collaboration-service/src/config/config.ts

```diff
@@ -0,0 +1,18 @@
+export const SUPABASE_URL = process.env['SUPABASE_URL'] as string;
+export const SUPABASE_KEY = process.env['SUPABASE_KEY'] as string;
+
+if (!SUPABASE_URL || !SUPABASE_KEY) {
+  console.error('FATAL ERROR: Missing required environment variables');
+  console.error('Required: SUPABASE_URL, SUPABASE_KEY');
+  console.error('Please check your .env file');
+  throw new Error('Missing required environment variables: SUPABASE_URL, SUPABASE_KEY');
+}
+
+export const JWT_SECRET = new TextEncoder().encode(process.env['JWT_SECRET'] || '');
+
+if (!JWT_SECRET) {
+  console.error('FATAL ERROR: Missing required environment variables');
+  console.error('Required: JWT_SECRET');
+  console.error('Please check your .env file');
+  throw new Error('Missing required environment variables: JWT_SECRET');
+}
```

Comment:
Perhaps could change this JWT check to keep JWT_SECRET as a string or check the length after encoding to prevent token authentication from silently using an empty key.
As an empty Uint8Array is still truthy (result of TextEncoder). So the "missing JWT_SECRET" condition will not trigger and an empty secret may be used!


```suggestion
// Option 1: Check the string before encoding
if (!process.env['JWT_SECRET']) {  // Checks if string is falsy (undefined or empty)
  throw new Error('Missing required environment variables: JWT_SECRET');
}
export const JWT_SECRET = new TextEncoder().encode(process.env['JWT_SECRET']);

// Option 2: Check the length after encoding
export const JWT_SECRET = new TextEncoder().encode(process.env['JWT_SECRET'] || '');

if (JWT_SECRET.length === 0) {  // Checks if array is empty
  throw new Error('Missing JWT_SECRET');
}
```

Reply:
nice catch! updated!